<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtServerApplicationClass</class>
 <widget class="QMainWindow" name="QtServerApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtServerApplication</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>191</width>
         <height>701</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>网络设置</string>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>25</y>
             <width>72</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>本地主机端口</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtPort">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>133</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>8080</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnStart">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>75</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>196</x>
         <y>9</y>
         <width>801</width>
         <height>691</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>数据日志</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTextEdit" name="txtDataLog"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>数据发送</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="txtDataSend"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnSend">
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtServerApplication.qrc"/>
 </resources>
 <connections/>
</ui>
